diff -Naur a/src/pkg_testing_tool/main.py b/src/pkg_testing_tool/main.py
--- a/src/pkg_testing_tool/main.py	2021-07-11 21:31:18.000000000 +0300
+++ b/src/pkg_testing_tool/main.py	2021-07-12 16:22:32.567105904 +0300
@@ -142,7 +142,7 @@
     emerge_cmdline = [
         'emerge',
         '--verbose', 'y',
-        '--autounmask', 'n',
+        '--oneshot',
         '--usepkg-exclude', job['cp'],
         '--deep', '--backtrack', '300',
     ]
diff -Naur a/src/pkg_testing_tool/use.py b/src/pkg_testing_tool/use.py
--- a/src/pkg_testing_tool/use.py	2021-07-11 21:31:18.000000000 +0300
+++ b/src/pkg_testing_tool/use.py	2021-07-12 16:23:41.160041342 +0300
@@ -39,10 +39,11 @@
 
     ignore_flags = set([
             'debug',
-            'doc',
+            'profile',
             'test',
             'selinux',
             'split-usr',
+            'systemd',
             'pic'
     ])
 
