diff -Naur a/src/pkg_testing_tool/use.py c/src/pkg_testing_tool/use.py
--- a/src/pkg_testing_tool/use.py	2021-07-11 21:31:18.000000000 +0300
+++ c/src/pkg_testing_tool/use.py	2021-07-12 16:24:50.576305230 +0300
@@ -40,9 +40,13 @@
     ignore_flags = set([
             'debug',
             'doc',
+            'examples',
+            'gtk-doc',
+            'profile',
             'test',
             'selinux',
             'split-usr',
+            'systemd',
             'pic'
     ])
 
