diff -Naur a/src/pkg_testing_tool/main.py b/src/pkg_testing_tool/main.py
--- a/src/pkg_testing_tool/main.py	2021-07-11 21:31:18.000000000 +0300
+++ b/src/pkg_testing_tool/main.py	2022-11-24 11:24:08.869398546 +0200
@@ -242,6 +242,9 @@
     else:
         use_combinations = None
 
+    if args.max_use_combinations == 1:
+        use_combinations = None
+
     if use_combinations:
         if package_metadata['has_tests'] and args.test_feature_scope == 'always':
             test_feature_toggle = True
